torch/utils/_config_module.py

```diff
@@ -123,6 +150,15 @@ class _ConfigEntry:
     # The value specified by the user when they overrode the configuration
     # _UNSET_SENTINEL indicates the value is not set.
     user_override: Any = _UNSET_SENTINEL
+    # The justknob to check for this config
+    justknob: Optional[str] = None
+
+    def __init__(self, config: Any):
+        if not isinstance(config, Config):
+            self.default = config
```

Comment:
It's probably better to just for call sites to always pass in Config. This is an internal ctor anyway

Reply:
done.